doc/module-docs/cc_package_update_upgrade_install/data.yaml

```
@@ -9,5 +9,9 @@ cc_package_update_upgrade_install:
   - comment: |
       Example 1:
     file: cc_package_update_upgrade_install/example1.yaml
+  - comment: "By default, `package_upgrade: true` will trigger upgrades on any installed package manager. To avoid calling `snap refresh` in images with snap installed, setting snap refresh.hold to a time limit to prevent cloud-init interaction during first boot"
```

Tiny grammar nit for clarity.

```suggestion
  - comment: "By default, `package_upgrade: true` will trigger upgrades on any installed package manager. To avoid calling `snap refresh` in images with snap installed, set snap refresh.hold to a time limit to prevent cloud-init interaction during first boot"
```

For the sentence in question, the second phrase is acting as a modifier to the first dependent clause, and thus needs to be an independent clause, but due to the "-ing", it becomes a dependent clause too. :P

---

thanks for the catch @a-dubs